test/fixtures/action_text/rich_texts.yml

```diff
@@ -0,0 +1,4 @@
+# one:
```

Comment:
We probably should have a fixture for a rich text record of a page.

Reply:
This file was is moved to fixtures/action_text/rich_texts.yml. Change will be reflected when pr fixes are merged.